experimental/ModelBuilder/test/BUILD

```diff
@@ -31,16 +31,16 @@ iree_lit_test_suite(
     data = [
         # runtime libraries
         "@llvm-project//mlir:tools/libvulkan-runtime-wrappers.so",
-        ":runtime-support.so",
+        "//experimental/ModelBuilder/test:runtime-support.so",
```

Comment:
I think this shouldn't be necessary. We should handle local references fine in bazel_to_cmake. Is it not working?

Reply:
You're right. Looking at it again it wasn't working due to the colon at the beginning. This is not needed in bazel so I removed it.

Reply:
Hmm we have special handling for the leading colon, so this should work https://sourcegraph.com/github.com/google/iree@6b7b71bdcf024caf8b302b06a9ee608f9126a997/-/blob/build_tools/bazel_to_cmake/bazel_to_cmake_converter.py#L202
https://sourcegraph.com/github.com/google/iree/-/blob/build_tools/cmake/iree_cc_library.cmake#L83

Reply:
What issue are you seeing with bazel_to_cmake if you leave this unchanged? My understanding is that this should work fine with leading colons in BUILD getting translated to leading `::` in CMakeLists.txt, which we then prepend with the package path. I'm curious what failed.